travis.sh

```diff
@@ -3,6 +3,9 @@ set -e #break script on non-zero exitcode from any command
 gem install bundler
 bundle install
 
+CTEST_OUTPUT_ON_FAILURE=ON
+export CTEST_OUTPUT_ON_FAILURE
```

Comment:
No, that's a short hand in bash and other korn shell derivatives. Bourne shell (/bin/sh) doesn't support that.